var/spack/repos/builtin/packages/rocm-openmp-extras/package.py

```diff
@@ -217,7 +252,8 @@ def patch(self):
                 aomp_extras.format(src) + '/libm/CMakeLists.txt')
 
         # Openmp adjustments
-        if self.spec.version >= Version('4.5.0'):
+        if (self.spec.version >= Version('4.5.0') or
+            self.spec.version == Version('master')):
```

Comment:
Correct me if I am wrong, but Spack know that master > 4.5.0. From my understanding Spack knows that:
```
develop > master > any_numbered_version
```

Reply:
made necessary changes